models/classes/runner/session/TestSession.php

```diff
@@ -145,14 +145,27 @@ public function getItemTags(RouteItem $routeItem)
         ];
 
         if ($this->isRunning() === true) {
-            $itemSession = $this->getAssessmentItemSessionStore()->getAssessmentItemSession($itemRef, $occurrence);
-            $tags[] = $itemId . '#' . $occurrence . '-' . $itemSession['numAttempts']->getValue();
+            $tags[] = $this->getItemAttemptTag($routeItem);
         }
 
         return $tags;
     }
 
     /**
+     * Gets the item tags for its last occurrence
+     * @param RouteItem $routeItem
+     * @return string
+     */
+    public function getItemAttemptTag(RouteItem $routeItem)
```

Comment:
Yes, it is. I added this method to factorise the naming format of the tag targeting the current attempt. It is needed by the QTI runner service to retrieve the duration of the current attempt, as you can see here: https://github.com/oat-sa/extension-tao-testqti/pull/469/files#diff-b75becba4988d98fae46d66f804e3699R334